apollo-federation/tests/query_plan/build_query_plan_tests/requires/include_skip.rs

```diff
@@ -230,167 +228,11 @@ fn it_handles_an_at_requires_where_multiple_conditional_are_involved() {
                         }
                       },
                     },
-                  }
+                  },
                 },
-              }
-            },
-          }
-        "###
-    );
-}
-
-#[test]
-fn unnecessary_include_is_stripped_from_fragments() {
```

Comment:
Should these two tests be removed?

Reply:
Good catch, added the tests back and they still pass, it should be fine to keep them.